scripts/devops_tasks/setup_execute_tests.py

```diff
@@ -21,6 +21,7 @@
 # a return code of 5 from pytest == no tests run
```

Comment:
If we want to allow no tests and exit code 5 only means that no tests were run, seems so from the docs (https://github.com/pytest-dev/pytest/blob/master/doc/en/usage.rst#possible-exit-codes),  then I'd suggest we simply handle that return code and not fail, as opposed to introducing a fake test.

Reply:
Yeah. I have come to this same conclusion. 

The method I'm going for:

in setup_execute_tests.py:
* If mgmt plane is in the package name being built, allow return code 5 (we do not want to allow this in general)
* allow coverage `xml` to fail
* set continueOnError to 1 for `publish coverage`

This is a compromise. We do not allow error code 5 in general, but allow exceptions for management plane. We will **still see** that the build was iffy due to the fact that it will be highlighted yellow in devops.